src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -217,6 +227,22 @@ the event.  For instance if an exception was captured the hint might carry the
 original exception object.  Not all SDKs are required to provide this.  The
 parameter however is reserved for this purpose.
 
+## Event Pipeline
+
+Any event that is captured by `capture_event` is processed in the following
+order and can be discarded at any of the stages, at which point no further
+processing happens.
+
+1. If the SDK is disabled or there is no active client, the event is discarded
```

Comment:
Given the definition of `disabled SDK` above, isn't `SDK is disabled or there is no active client` redundant?

Reply:
agreed